src/firebaseConnect.js

```diff
@@ -62,7 +62,7 @@ export const createFirebaseConnect = (storeKey = 'store') =>
         componentWillReceiveProps (np) {
           const { firebase, dispatch } = this.store
           const inputAsFunc = createCallable(dataOrFn)
-          const data = inputAsFunc(np, firebase)
+          const data = inputAsFunc(np, this.store)
```

Comment:
Whats this change about?

Reply:
To provide `store` as the second argument to functions passed to `firebaseConnect`. That way, `store.getState` can be used without needing to use a preceding `connect` call.

Reply:
It is mentioned in [the migration guide](http://docs.react-redux-firebase.com/history/v2.0.0/docs/v2-migration-guide.html#stateBasedQueries)

Reply:
Ok - seems like it would be breaking if you were expecting firebase as the second argument.  We don't use a second argument anywhere, so it doesn't matter to us

Reply:
Yup, totally agree. It will be mentioned as a breaking change in the release. It is still available as `store.firebase`, but I was also thinking of making a config option that would allow users to keep things as they were.